Resources/public/ts/StingerSoftAggrid.ts

```diff
@@ -110,7 +110,8 @@ export class StingerSoftAggrid {
         return jQuery.getJSON(this.options.stinger.ajaxUrl, {
             'agGrid': {
                 'gridId': '#' + that.gridId
-            }
+            },
+            'extraData': this.options['extraData']
```

Comment:
This also doesn't sound like "data" is given. Should be renamed to something more meaningful